app/src/main/java/com/cornellappdev/android/eatery/MapsActivity.java

```diff
@@ -101,15 +102,10 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                 startActivity(intent);
                 break;
               case R.id.action_brb:
-                Snackbar snackbar =
-                    Snackbar.make(
-                        findViewById(R.id.maps_activity),
-                        "If you would like"
-                            + " to see this feature, consider joining our Android dev team!",
-                        Snackbar.LENGTH_LONG);
-                snackbar.setAction("Apply", new SnackBarListener());
-                snackbar.show();
-
+                finish();
+                intent = new Intent(getApplicationContext(), InfoActivity.class);
+                intent.putExtra("cafeData", cafeData);
```

Comment:
"cafeData" should go into strings.xml as well

Reply:
I don't think intent identifiers belong in the strings.xml file. When I tried to use a string from strings.xml, the line threw an error

Reply:
ye we should either create a final static string, not put it in the strings.xml. It wouldn't work.